lib/arel_extensions/visitors/mssql.rb

```diff
@@ -270,7 +270,9 @@ def visit_ArelExtensions_Nodes_Format o, collector
         if fmt = LOADED_VISITOR::DATE_CONVERT_FORMATS[f]
           collector << "CONVERT(VARCHAR(#{f.length})"
           collector << LOADED_VISITOR::COMMA
+          collector << 'CONVERT(datetime' << LOADED_VISITOR::COMMA << ' ' if !o.time_zone.empty?
```

Comment:
While in practice it will always be the case, from a dependency point of view, because we don't depend on Rails, we should not expect present? to be... present.